wiki_trac_rst_convert.py

```diff
@@ -0,0 +1,74 @@
+# Convert local files formated in Trac Wiki RST to Vanilla RST.
+import re
+import sys
+import os
+
+from wiki_migrate import get_page_name
+
+
+def main():
+    """
+    Do the job.
+    """
+    if len(sys.argv) != 2:
+        print("Need to pass the path to wiki base directory.")
+        sys.exit(1)
+
+    for root, _, files in os.walk(sys.argv[1]):
+        for name in files:
+            convert_file(os.path.join(root, name))
+
+    print('Conversion complete.')
+
+
+def convert_file(path):
+    """
+    In-place conversion of files; no backup.
+    """
+    print('Converting ', path)
+    with open(path) as f:
+        text = f.read()
+
+    with open(path, 'w') as f:
+        f.write(convert_content(text))
+
+
+def convert_content(text):
+    """
+    Conver from Trac wiki RST format to standard RST format.
+    """
+    # Remove RST wrapping.
+    result = text.strip('}}}')
+    result = result.strip('{{{')
+    result = result.strip()
+    result = result.replace('#!rst', '')
+    result += '\n'
+    result = _trac_wiki_to_plain_links(result)
+
+    return result
+
+
+def _trac_wiki_to_plain_links(text: str):
+    """
+    Converts Trac wiki links to plain relative links
+    :param result:
+    :return:
```

Comment:
Is nice to use type annotation for python3.


But no need for Sphinx type API docs... with the current version that info (param and return) is confusing.

This is a simple script so we don't need to be to formal about the API.

But the docstring can be a bit better

By reading `_trac_wiki_to_plain_links` I already read `convert Trac wiki to plain links`

So the docstring can explain a bit more.

It looks like this only updates the Trac RST wiki link directives.

In TracWiki format (not RST with Trac directives) we also have these links:

```
 * [wiki:Administrative Project management and administration] 
 * [https://chevah.com/buildbot/ Buildbot]
 ```

So the docstring can be something like that:

"""
It takes RST conttent with Trac wiki links directives and coverts the link directives to single line RST vanilla links.
"""

We don't want to use footnotes links. They are harder to work with.

The plan is to convert the content and then to continue to edit the content using a text editor or GitHub Wiki GUI.